meson.build

```diff
@@ -0,0 +1,112 @@
+project(
+  'afpfs-ng',
+  'c',
+  version: '0.8.2',
+  license: 'GPLv2',
+  default_options: ['warning_level=3', 'c_std=gnu99']
+)
```

Comment:
Consider adding

```meson
    meson_version: '>=0.60.0',
```

or some other meson version range to target.

If you try to go lower than 0.60, meson prints this:


> meson.build:85: WARNING: Project targets '>=0.44.0' but uses feature introduced in '0.60.0': dep 'iconv' custom lookup. consider checking for `iconv_open` with and without `find_library('iconv')`

Reply:
0.61 is a good baseline since it brought install_symlink(). We don't use features from later version, yet.